hype-submitter_2.11/src/main/scala/com/spotify/hype/GkeCluster.scala

```diff
@@ -0,0 +1,16 @@
+package com.spotify.hype
+
+
+case class GkeCluster(project: String,
+                      zone: String,
+                      cluster: String,
+                      staging: String
```

Comment:
Staging is not really part of the GKE cluster so I do not think it belongs in this object.